api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -126,11 +126,31 @@ message ExternalProcessor {
   config.core.v3.GrpcService grpc_service = 1
       [(udpa.annotations.field_migrate).oneof_promotion = "ext_proc_service_type"];
 
-  // [#not-implemented-hide:]
   // Configuration for the HTTP service that the filter will communicate with.
   // Only one of ``http_service`` or
   // :ref:`grpc_service <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.grpc_service>`.
   // can be set. It is required that one of them must be set.
+  //
+  // If ``http_service`` is set, the
+  // :ref:`processing_mode <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExternalProcessor.processing_mode>`
+  // can not be configured to send any body or trailers. i.e, http_service only supports
+  // sending request or response headers to the side stream server.
+  //
+  // With this configuration, Envoy behavior:
+  // 1. The headers are first put in a proto message
+  // :ref:`ProcessingRequest <envoy_v3_api_msg_service.ext_proc.v3.ProcessingRequest>`.
+  // 2. This proto message is then transcoded into a JSON text.
+  // 3. Envoy then sends a HTTP POST message with content-type as "application/json",
+  // and this JSON text as body to the side stream server.
+  //
+  // After receives this HTTP request message, the side stream server behavior:
+  // 1. It converts the body, which is a JSON string, into a ``ProcessingRequest`` proto message.
+  // 2. It performs whatever header mutation on the headers in this ``ProcessingRequest``.
```

Comment:
These two lines number are duplicated and they can be merged into 1:
It can convert the body, which is a JSON string, into a ``ProcessingRequest`` proto message to examine the headers.